index.html

```diff
@@ -27,19 +34,18 @@
   </head>
   <body>
     <div class="first-page">
-      <header class="header">
-        <div class="container">
-          <a href=""><img src="images/MyLogo.png" alt="Logo" class="logo" /></a>
-          <nav class="item-2">
-            <img src="images/Hamburguer.png" alt="Menu" />
-            <ul class="ul-menu">
-              <li>Portfolio</li>
-              <li>About</li>
-              <li>Contact</li>
-            </ul>
-          </nav>
-        </div>
+      <header class="navbar">
+        <a href="#" class="logo"><img src="images/MyLogo.png" alt="" /></a>
+        <div id="icon" class="icon fas fa-bars"></div>
+        <nav class="ul-menu">
+          <ul class="o">
+            <li class="desktop"><a href="#portfolio">Portfolio</a></li>
+            <li class="desktop"><a href="#about">About</a></li>
+            <li class="desktop"><a href="#contact">Contact</a></li>
+          </ul>
```

Comment:
One last thing is... 
- [ ] I can barely see your `X` button when I open the Pop-Up menu. I would suggest you make it more visible to users and on your each list items have list-style `.` which is makes your Pop-Up menu different from the original design I would strongly suggest you to this as well. 

As you can see in this image...

<img width="366" alt="Screen Shot 2022-01-25 at 7 37 30 AM" src="https://user-images.githubusercontent.com/69011963/150904061-c3dabe64-fbaa-400f-90d6-033d9019754b.png">